Authors.md

```diff
@@ -12,5 +12,5 @@ Orlando Hohmeier | orlando@mesosphere.com | Current Maintainer
 Pierluigi Cau | pierlo@mesosphere.com | Current Maintainer
 Philipp Hinrichsen | philipph@mesosphere.com | Current Maintainer
 Philip Norman | philip@mesosphere.com | Current Maintainer
-Felix Gertz | felix@mesosphere.com | Current Maintainer
+Felix Gertz | https://github.com/aldipower | Contributor
```

Comment:
I really don't feel like adding personal emails either. I think the github profile makes sense as it gives a way to get in touch with the contributor.